include/flatbuffers/reflection.h

```diff
@@ -503,6 +510,10 @@ bool Verify(const reflection::Schema &schema, const reflection::Object &root,
             const uint8_t *buf, size_t length, uoffset_t max_depth = 64,
             uoffset_t max_tables = 1000000);
 
+bool VerifySizePrefixed(const reflection::Schema &schema, const reflection::Object &root,
```

Comment:
you can drop the reflection:: qualifier

Reply:
VerifySizePrefixed is declared in the flatbuffers namespace and Schema and Object are in the reflection namespace - see Verify.

Reply:
Ah, my bad. I just saw that it was in the reflection.h and assumed.